src/static_vars.h

```diff
@@ -119,6 +130,8 @@ class Static {
     uintptr_t extra;  // To force alignment
   };
   ATTRIBUTE_HIDDEN static PageHeapStorage pageheap_;
+
+  static std::atomic<int64_t> sample_period_;
```

Comment:
Default value?

Reply:
Initialized in `InitStaticVars` in `static_vars.cc`.